src/QuadForm/Herm/Genus.jl

```diff
@@ -255,8 +255,12 @@ $\mathfrak p$-valuation.
 function norm(G::HermLocalGenus)
   p = prime(G)
   OK = order(p)
-  if !is_dyadic(G) || !is_ramified(G)
+  if !is_ramified(G)
     return fractional_ideal(OK, p)^scale(G, 1)
+  elseif !is_dyadic(G)
+    # If the prime ideal ramifies, then p = P^2, so we need to divide the scale
+    # valuation by 2 to obtain the correct norm valuation
+    return fractional_ideal(OK, p)^(div(scale(G, 1), 2))
```

Comment:
I think that this is wrong. There are subnormal planes in the ramified non-dyadic case. 
Namely the hyperbolic plane should be subnormal. Here the norm is smaller than the scale.

Reply:
I looked at Markus habilitation and yes, I forgot about subnormal sums of planes. This it is a bit more than this: the blocks of even scale valuations are diagonal so we must divide the valuation, and the ones of odd valuation are subnormal and the valuation stays the same. Then we must take the minimum of these new valuations. I will push the fix once the tests pass.